taxonomies/categories.txt

```diff
@@ -27783,8 +27783,8 @@ ciqual_food_name:en:Yam or Indian potato, peeled, boiled/cooked in water
 ciqual_food_name:fr:Igname, épluchée, bouillie/cuite à l'eau
 
 <en:Yam potatoes
-en:Raw peeled yam potato, Raw peeled Indian potato
-fr:Igname épluchée crue
+en:peeled yam potato, peeled Indian potato
```

Comment:
this is an example of a ciqual category that has no sense for OFF, we should not create a separate category because it's peeled. (and it's probably almost never sold peeled)

Reply:
fixed. Moved as values for en:Yam potatoes